### PR TITLE
#8 - User management

### PR DESCRIPTION
**Pull request by mixxorz**
[Codebase ticket #8](https://projects.torchbox.com/projects/flare-rebuild/tickets/8)

#### When applied this MR will...

Add user authentication and user management to FLARe Portal

#### Screenshots

localhost_8000_accounts_login__next=_
localhost_8000_users_2_
localhost_8000_users_

#### Checklist

- [x] PR addresses all ACs
- [x] Added unit tests if necessary
- [x] Updated documentation if necessary
- [ ] CI passes
- [ ] Code reviewed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,5 @@ wheels/
 *.egg-info/
 .installed.cfg
 *.egg
+.vim
+.nvimrc
```
Comment (NathHorrigan) [inline .gitignore:38]: Heh, Didn't know you used Vim. I've tried a few times, Never succeeded.

File: flare_portal/users/models.py [hunk 2]
```diff
@@ -1,5 +1,77 @@
 from django.contrib.auth.models import AbstractUser
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+
+from . import constants
 
 
 class User(AbstractUser):
```
Comment (NathHorrigan) [inline flare_portal/users/models.py:8]: Nice User Model!

**Comment by NathHorrigan** [comment]
LGTM!

File: flare_portal/users/models.py (hunk 2 above)
Comment (mixxorz) [reply flare_portal/users/models.py:8]: This comes with Wagtail-Kit 😛

File: .gitignore (hunk 1 above)
Comment (mixxorz) [reply .gitignore:38]: I literally cannot use any other text editor now lol.

**Comment by mixxorz** [comment]
Thanks Nathan! Merging this in.

**Pull request merged**